app/services/methodology_migration_service.rb

```diff
@@ -0,0 +1,102 @@
+class MethodologyMigrationService
+  attr_reader :project
+
+  def initialize(project_id)
+    @project = Project.find(project_id)
+  end
+
+  def migrate(methodology, board_name: nil, node: nil)
+    ActiveRecord::Base.transaction do
+      # create a board for each methodology
+      board = Board.new(
+        name: methodology.name,
+        node: node || project.methodology_library
+      )
```

Comment:
I am not 100% sure we must remove the `project_id: project_id` line here.
If we replace it with `project: @project`, the line will be the same in both codebases.